README.md

```diff
@@ -117,6 +117,13 @@ chatkit_integration_locator=<INSTANCE_LOCATOR>
 chatkit_integration_key=<SECRET_KEY>
 ```
 
+## Linting
+
+We currently use [ktlint](https://github.com/pinterest/ktlint). You can use `ktlintCheck` to check if there are any errors. Most errors can be automatically resolved by using `ktlintFormat` - if it can't auto resolve the issue, it will let you know what to do.
```

Comment:
```suggestion
We currently use [ktlint](https://github.com/pinterest/ktlint).
You can use `ktlintCheck` to check if there are any errors.
Most errors can be automatically resolved by using `ktlintFormat` — if it can't auto resolve the issue,
it will let you know what to do.
```
Em dash + breaking up a very long line.